src/index.html

```diff
@@ -5,8 +5,388 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap"
+          rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="moyo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphone" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+            href="#laptops"
+            data-qa="nav-hover"
+            class="nav__link"
+            >
+            Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadget" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablet" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#photo" class="nav__link">Photo</a>
          </li>

          <li class="nav__item">
            <a href="#video" class="nav__link">Video</a>
          </li>
```
to increase readability add a blank line between **sibling** elements (fix everywhere)